tests/test_options.py

```diff
@@ -146,9 +146,9 @@ def test_multiple_default_help(runner):
     @click.command()
     @click.option("--arg1", multiple=True, default=('foo', 'bar'),
                   show_default=True)
-    @click.option("--arg2", multiple=True, default=(1, 2), type=int,
+    @click.option("--arg2", multiple=True, default=(1, 2), type=(int,int),
```

Comment:
Ok. If I remember correctly I changed this just because I saw it and thought it might be a typo.
I don't know why click even accept this contradictory options.
It might not be the best thing to include it in this pull request, sry.